environment.yml

```diff
@@ -0,0 +1,7 @@
+name: amazon-study
```

Comment:
is this file redundant with ENV.yml?
How are they different?
What are each of them for?